admin/json.go

```diff
@@ -21,6 +21,11 @@ func writeJSONError(w http.ResponseWriter, err *HttpError) {
 	writeJSON(w, response)
 }
 
+func writeJSONOk(w http.ResponseWriter) {
+	response := make(map[string]string)
```

Comment:
Apparently you can't make a map a constant, haven't found a good way to make this whole response one.